.claude/skills/firebase-prod-campaigns.md

```
@@ -257,11 +257,11 @@ From `$PROJECT_ROOT/llm_service.py`:
 - **Max history turns:** 100 (truncation limit)
 - **200K threshold:** Above this = "long context" pricing (2x cost)
 
-### Gemini 3 Pro Pricing
+### Gemini 3 Flash Pricing
 | Context Size | Input | Output |
```

markdownlint-cli2 (0.20.0)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)